index.bs

```diff
@@ -388,7 +388,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 3. Set |callerOrigin| to the <a>origin</a> of the caller. Derive the RP ID from |callerOrigin| by computing the
     "public suffix + 1" or "PS+1" (which is also referred to as the "Effective Top-Level Domain plus One" or "<a>eTLD+1</a>")
-    part of |callerOrigin| [[PSL]]. Set |rpId| to the RP ID.
+    part of |callerOrigin| [[PSL]]. Set |rpIdHash| to the SHA-256 hash of the UTF-16 encoding of the lowercase form of this RP
```

Comment:
OK. I picked UTF-16 because that's what DOMString is, and to hopefully make i18n easier, but I don't have an issue with UTF-8 either.